module/module.go

```diff
@@ -305,12 +305,17 @@ func (moduleSystem *System) GenerateBuild(
 				targetGenDir,
 				classInstance.Directory,
 			)
+			prettyBuildPath := filepath.Join(
```

Comment:
You want path.Rel probably. Can't assume that Base is one level relative to the cwd.